src/drafty.js

```diff
@@ -1582,6 +1763,136 @@ Drafty.getContentType = function() {
   return 'text/x-drafty';
 }
 
+// Removes reply quotes and any text/formatting associated with these.
+// Returns a Drafty document free of reply quotes.
+function stripQuotes(original) {
```

Comment:
I think this is excessively complex and partially replicates existing code. The idea here is to convert Drafty to a tree of spans, then to prune branches. `toTree` already converts span array to a tree.

If you want, I can take a shot at this.

Reply:
Sure.